sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1651,6 +1652,34 @@ export class ApplicationService extends RecordDataModelService<Application> {
     }
   }
 
+  /**
+   * Gets application and assessment status details.
+   * @param applicationId application id.
+   */
+  async getApplicationAssessmentStatusDetails(
+    applicationId: number,
+  ): Promise<ApplicationAssessmentStatusDetailsAPIOutDTO> {
```

Comment:
Controller DTOs are not supposed to be returned from the services.
Services can return the data and controller or controller service will adapt them.